versions.yaml

```diff
@@ -104,6 +104,15 @@ assets:
           branch: "master"
           tag: "v3.1.0-rc2"
           commit: "47c1cc30e440860aa695358f7c2dd0b9d7b53d16"
+      hyperv:
+        description: |
+          Due to a missing KVM feature in hyperV is not possible to run
```

Comment:
OOI, is there a URL we can add here for a link to further details on the status of this missing feature in HyperV?